versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
```

Comment:
this is a breaking change, should be a major bump unless you set "=> 4.0, < 6.0"

Reply:
@bushong1 thank you for pointing this out. Configured the CHANGELOG to reflect this with a major version bump.